### DISCUSSION TITLE
Allow billing notification banners to be dismissed

### DISCUSSION THREAD

**Discussion by mway** [community/community#49459] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi friends!

One of the orgs my company owns is having a billing issue, and while I'm an owner, I'm not responsible for billing and would like to stop seeing the banner at the top of every page I look at on GH.

Specifically, I'm referring to this:

<img width="2038" alt="image" src="https://user-images.githubusercontent.com/61755/223572529-51c36136-ffe7-4af3-b846-6853c5a39b7d.png">

That's a lot of real estate. I'm sure the billing issue will be resolved, but again, just because I'm an admin doesn't mean I can do anything about it, so it mostly just serves as an ineffectual nag in this case.

Another alternative would be moving this to email. In my company, the people who are responsible for billing are probably not using GH, so they will never see this banner; but if you were to send them an email (even a recurring one), that would probably be more actionable. Or maybe make it configurable? Just spitballing.

At a minimum, any chance we can get a dismiss button?

Thanks for your consideration!

**Comment by adamkdean** [comment]
I agree with this. It takes up a lot of space and for those of us who are across several, sometimes dozens of orgs, it only takes one to miss a payment and we have to see the banner every day until resolved (as you say @mway, not always our responsibility). What alternative is there, to leave the org? At the very least allow it to be minimised. cc @queenofcorgis

**Comment by axsms** [comment]
This and the multiple notification emails that tells us our included actions minutes are about to run out, while we have a financial limit set as a backup, is a distraction and annoying. I wasn't even aware that our employees see the banner until I read the Feedback and comment. That seems unhelpful in our situation.
